### PR TITLE
Zerotier join game

### LINKED ISSUE #4186: Join game screen should be ZeroTier aware
The Join screen currently assumes TCP is the only connection method:

image

- The headline should be changed to: "Join ZeroTier Games"
- Description should read "Enter a Game ID to join a game already in progress"
- A different addressbook entry in the ini should be used for the input memory

### PR DESCRIPTION
**Pull request by AJenbo**
Fixes #4186

### CODE REVIEW

File: Source/options.cpp [hunk 1]
```diff
@@ -327,6 +327,7 @@ void LoadOptions()
 	GetIniValue("Hellfire", "SItem", sgOptions.Hellfire.szItem, sizeof(sgOptions.Hellfire.szItem), "");
 
 	GetIniValue("Network", "Bind Address", sgOptions.Network.szBindAddress, sizeof(sgOptions.Network.szBindAddress), "0.0.0.0");
+	GetIniValue("Network", "Previous Host", sgOptions.Network.szPreviousZTGame, sizeof(sgOptions.Network.szPreviousZTGame), "");
```
Comment (qndel) [inline Source/options.cpp:330]:
```suggestion
	GetIniValue("Network", "Previous Game ID", sgOptions.Network.szPreviousZTGame, sizeof(sgOptions.Network.szPreviousZTGame), "");
```

File: Source/DiabloUI/selgame.cpp [hunk 2]
```diff
@@ -303,13 +311,20 @@ void selgame_GameSelection_Select(int value)
 		break;
 	}
 	case 2: {
-		title = _("Join TCP Games").c_str();
+		title = fmt::format(_("Join {:s} Games"), _(ConnectionNames[provider])).c_str();
+
+		std::string intputHint;
```
Comment (qndel) [inline Source/DiabloUI/selgame.cpp:316]:
```suggestion
		std::string inputHint;
```

Comment (AJenbo) [reply Source/DiabloUI/selgame.cpp]: Fixed, and also tested now :P

**Review by qndel — APPROVED** [review]
Approved!

**Comment by AJenbo** [comment]
Thanks :)

**Pull request merged**